src/index.html

```diff
@@ -11,7 +11,138 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <!-- <h1>HTML Form</h1> -->
     <script type="text/javascript" src="./main.js"></script>
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post">
+      <fieldset class="block-1">
+        <legend>Personal information</legend>
+        <label>
+          Surname:
+          <input
+            type="text"
+            autocomplete="off"
+          >
+        </label>
+        <label>
+          Name:
+          <input
+            type="text"
+            autocomplete="off"
+          >
+        </label>
+        <label>
+            How old are You?
+          <input
+            value="12"
+            type="number"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label>
+          Full date of birth:
+          <input type="date">
+        </label>
+        <label>
+          I accept the term of the agreement
+          <input type="checkbox">
+        </label>
+      </fieldset>
+      <fieldset class="block-2">
+        <legend>Registration</legend>
+        <label>
+          E-mail:
+          <input
+            type="email"
+            placeholder="email@example.com"
+          >
+        </label>
+        <label>
+          Password:
+          <input
+            type="password"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block-3">
+        <legend>
+          An interesring fact about you
+        </legend>
+        <div
+          id="love-cats">
+          Do you love cats?
+          <label>
+            <input
+              type="radio"
+              name="choose"
+            >
+              Yes
+          </label>
+          <label>
+            <input
+              type="radio"
+              name="choose"
+            >
+              No
+          </label>
+        </div>
+        <label>
+          What is your favorite color?
+          <input
+            type="color"
+          >
+        </label>
+        <label>
+          What time do you go to bed?
+          <input
+            type="time"
+          >
+        </label>
+        <label>
+          What are your favorite brand of cars?
+          <select multiple required>
+            <option value="bmw">BMW</option>
+            <option value="audi">Audi</option>
+            <option value="lada">Lada</option>
+          </select>
+        </label>
+        <label>
+          How do you rate our work?
+          <input
+            type="range"
+            value="0"
+          >
+        </label>
+      </fieldset>
+      <fieldset class="block-4">
+        <legend>Additional info:</legend>
+        <label
+          for="comments">
+          Comments:
+          <textarea
+            name="comments"
+            id="comments"
+            cols="20"
+            rows="2">
+          </textarea>
```

Comment:
```suggestion
          <textarea
            name="comments"
            id="comments"
            cols="20"
            rows="2"></textarea>
```
To avoid 
![image](https://user-images.githubusercontent.com/97547824/201763543-06d7279e-bb4c-4719-ac48-eebfb7421f94.png)